api/src/test/java/bio/terra/tanagra/aousynthetic/DeviceEntityQueries.java

```diff
@@ -0,0 +1,60 @@
+package bio.terra.tanagra.aousynthetic;
+
+import static bio.terra.tanagra.aousynthetic.UnderlayUtils.ALL_DEVICE_ATTRIBUTES;
+import static bio.terra.tanagra.aousynthetic.UnderlayUtils.BQ_DATASET_SQL_REFERENCE;
+import static bio.terra.tanagra.aousynthetic.UnderlayUtils.DEVICE_ENTITY;
+import static bio.terra.tanagra.aousynthetic.UnderlayUtils.UNDERLAY_NAME;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import bio.terra.tanagra.app.controller.EntityInstancesApiController;
+import bio.terra.tanagra.generated.model.ApiEntityDataset;
+import bio.terra.tanagra.generated.model.ApiGenerateDatasetSqlQueryRequest;
+import bio.terra.tanagra.generated.model.ApiSqlQuery;
+import bio.terra.tanagra.testing.BaseSpringUnitTest;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+/**
+ * Tests for device entity queries on the AoU synthetic underlay. There is no need to specify an
+ * active profile for this test, because we want to test the main application definition.
+ */
+public class DeviceEntityQueries extends BaseSpringUnitTest {
```

Comment:
nit: existing pattern in the codebase is to suffix tests with `Test` in the classname

Reply:
done.